packages/datadog-plugin-openai/test/integration-test/client.spec.js

```diff
@@ -43,9 +44,11 @@ describe('esm', () => {
         assert.strictEqual(checkSpansForServiceName(payload, 'openai.request'), true)
       })
 
-      proc = await spawnPluginIntegrationTestProc(sandbox.folder, 'server.mjs', agent.port)
+      proc = await spawnPluginIntegrationTestProc(sandbox.folder, 'server.mjs', agent.port, null, {
+        NODE_OPTIONS: '--import dd-trace/register.js'
+      })
 
       await res
-    }).timeout(20000)
+    }).timeout(5000)
```

Comment:
```suggestion
    }).timeout(20000)
```
oops from testing 20s was too long a wait 😅